startmicro

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+regex="([^\/]+)-([^-]+)-cache-config.yaml"
+node_yaml='(() => {
+var yaml = require("js-yaml");
+var fs   = require("fs");
+var doc  = yaml.safeLoad(fs.readFileSync(process.argv[1], "utf8"));
```

Comment:
Not sure what you mean. If it finds the cache-config.yaml file, it is passed to the "HERE-script" via `key_secret=$(node -p "$node_yaml" "$file")` below. The doc name is never omitted.